core/templates/dev/head/pages/story_editor/main_editor/story_node_editor_directive.html

```diff
@@ -1,70 +1,71 @@
-            ng-if="isOutlineModified(editableOutline)"
-            ng-click="updateOutline(editableOutline)">
-      Save
-    </button>
-    <button type="button"
-            class="btn btn-default save-button"
-            ng-if="!isOutlineModified(editableOutline)"
-            ng-click="openPreviewOutline(editableOutline)">
-      Preview
-    </button>
+  <div class="node-field">
+    <label> Next Chapters </label>
+    <md-card class="destination-node-id"
+             ng-repeat="nodeId in getDestinationNodeIds()"
+             ng-click="viewNodeEditor(nodeId)">
+      <[nodeIdToTitleMap[nodeId]]>
+      <span style="float: right;" ng-click="removeDestinationNodeId(nodeId)" aria-hidden="true">
+        <i class="material-icons md-18">&#xE14C;</i>
+      </span>
+    </md-card>
+    <md-card class="destination-node-id" ng-if="!nodeTitleEditorIsShown" ng-click="openNodeTitleEditor()">
+      + Add existing chapter.
+    </md-card>
+    <md-card class="destination-node-id" ng-if="nodeTitleEditorIsShown" ng-click="openNodeTitleEditor()">
+      <input type="text" ng-model="newDestinationNodeTitle" placeholder="Enter chapter title" autofocus>
+      <button class="btn btn-success" ng-click="addExistingDestinationNode(newDestinationNodeTitle)">
+        Save
+      </button>
+      <button class="btn btn-default" ng-click="closeNodeTitleEditor()">
+        Cancel
+      </button>
+    </md-card>
+    <md-card class="destination-node-id" ng-click="addNewDestinationNode()">
+      + Add a new chapter.
+    </md-card>
   </div>
-</div>
-<div class="node-field">
-  <label> Destination Nodes </label>
-  <md-card class="destination-node-id"
-           ng-repeat="nodeId in getDestinationNodeIds()"
-           ng-click="viewNodeEditor(nodeId)">
-    <[nodeId]>
-    <span style="float: right;" ng-click="removeDestinationNodeId(nodeId)" aria-hidden="true">
-      <i class="material-icons md-18">&#xE14C;</i>
-    </span>
-  </md-card>
-  <md-card class="destination-node-id" ng-if="!nodeIdEditorIsShown" ng-click="openNodeIdEditor()">
-    + Add existing node.
-  </md-card>
-  <md-card class="destination-node-id" ng-if="nodeIdEditorIsShown" ng-click="openNodeIdEditor()">
-    <input type="text" ng-model="newDestinationNodeId">
-    <button class="btn btn-success" ng-click="addExistingDestinationNode(newDestinationNodeId)">
-      Save
-    </button>
-    <button class="btn btn-default" ng-click="closeNodeIdEditor()">
-      Cancel
-    </button>
-  </md-card>
-  <md-card class="destination-node-id" ng-click="addNewDestinationNode()">
-    + Create a new destination node.
-  </md-card>
+  <label style="padding-top: 2em;"> Node Outline </label> <br>
+  Outline Finalized
+  <i ng-if="!isOutlineFinalized()" ng-click="finalizeOutline()" class="material-icons md-18">check_box_outline_blank</i>
+  <i ng-if="isOutlineFinalized()" ng-click="unfinalizeOutline()" class="material-icons md-18">check_box</i>
+  <schema-based-editor schema="OUTLINE_SCHEMA"
```

Comment:
I actually didn't intend to change the way the outline currently works in develop. We should keep it such that the RTE isn't always displayed and is only displayed while editing. In all other times, just the content of the outline should be shown. Could you revert it to that? If you keep the RTE editor open after clicking "Save" on the outline editor, it's not very clear that anything saved because there's no visual feedback. Instead, the RTE should close and we should just display the updated outline content.

Also, I believe Sean mentioned moving the outline section to be right after the chapter title.

Reply:
I kept that permanently open as the story notes section and similar RTEs throughout the editors (like subtopic page editor) also have permanently open RTEs. Should I have it closed here anyway?